src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -969,18 +993,28 @@ private List<UtxoStakeDescription> GetUtxoStakeDescriptionsSuitableForStaking(Li
             return res;
         }
 
-        private int GetBlocksToMaturity(UtxoStakeDescription utxoStakeDescription)
+        /// <summary>
+        /// Calculates blocks count till UTXO is considered mature for staking.
+        /// </summary>
+        /// <param name="utxoStakeDescription">The utxo stake description.</param>
+        /// <returns>How many blocks are left till UTXO is considered mature for staking.</returns>
+        private int GetBlocksCountToMaturity(UtxoStakeDescription utxoStakeDescription)
         {
             if (!(utxoStakeDescription.UtxoSet.IsCoinbase || utxoStakeDescription.UtxoSet.IsCoinstake))
                 return 0;
 
             return Math.Max(0, (int)this.network.Consensus.Option<PosConsensusOptions>().CoinbaseMaturity + 1 - this.GetDepthInMainChain(utxoStakeDescription));
         }
 
-        // Return depth of transaction in blockchain:
-        // -1  : not in blockchain, and not in memory pool (conflicted transaction),
-        //  0  : in memory pool, waiting to be included in a block,
-        // >=1 : this many blocks deep in the main chain.
+        /// <summary>
+        /// Gets depth of transaction in blockchain.
+        /// </summary>
+        /// <param name="utxoStakeDescription">The utxo stake description.</param>
+        /// <returns>
+        /// <c>-1</c> if not in blockchain, and not in memory pool (conflicted transaction).
+        /// <c>0</c> if in memory pool, waiting to be included in a block.
+        /// <c>>=1</c> if included in a block. Shows how many blocks deep in the main chain.
+        /// </returns>
```

Comment:
remember that `<` inside <code> that we replaced with &lt; ?
this `<=` should cause the same thing in your IDE

Reply:
For some reason if I use '&ge;' VS marks it as a mistake- undefined entity 'ge', but it's fine with >=. 
Though this is not fired with &lt; and &gt;
So are you ok with leaving it as it is?

Reply:
Actually we could use `Value greater than <c>0<c/> if included in a block.`
using `≥` character is fine for me, my VS does not mark it as invalid
but if you want, you can use `&gt;=`

Reply:
but TBH my VS does not complain about > or < or anything else, so you probably have something extra?

Reply:
Yeah, it's probably a warning from resharper or power tools.